src/Pages/uploadPage.js

```diff
@@ -0,0 +1,251 @@
+            uid: firebase.auth().currentUser.uid
+        }).then(() => {
+            alert('Submitted successfully');
+        })
+    };
+
+    updateInput = e => {
+        this.setState({
+            [e.target.name]: e.target.valueOf().checked
+        });
+    };
+
+    server = {
+        // this uploads the image using firebase
+        process: (fieldName, file, metadata, load, error, progress, abort) => {
+            // create a unique id for the file
+            const id = shortid.generate();
+
+            // upload the image to firebase
+            const task = storage.child('images/' + id).put(file, {
+                contentType: 'image/jpeg',
+            });
+
+            // monitor the task to provide updates to FilePond
+            task.on(
+                firebase.storage.TaskEvent.STATE_CHANGED,
+                snap => {
+                    // provide progress updates
+                    progress(true, snap.bytesTransferred, snap.totalBytes)
+                },
+                err => {
+                    // provide errors
+                    error(err.message)
+                },
+                () => {
+                    // the file has been uploaded
+                    load(id);
+
+                    let tempThis = this;
+
+                    storage.child('images/' + id).getDownloadURL().then(function (url) {
+                            console.log(url);
+                            tempThis.setState({image: url});
+                        }
+                    );
+                }
+            )
+        },
+
+        // this loads an already uploaded image to firebase
+        load: (source, load, error, progress, abort) => {
+            // reset our progress
+            progress(true, 0, 1024);
+
+            // fetch the download URL from firebase
+            storage
+                .child('images/' + source)
+                .getDownloadURL()
+                .then(url => {
+
+
+                    // fetch the actual image using the download URL
+                    // and provide the blob to FilePond using the load callback
+                    let xhr = new XMLHttpRequest();
+                    xhr.responseType = 'blob';
+                    xhr.onload = function (event) {
+                        let blob = xhr.response;
+                        load(blob)
+                    };
+                    xhr.open('GET', url);
+                    xhr.send()
+                })
+                .catch(err => {
+                    error(err.message);
+                    abort()
+                })
+        },
+    };
+
+    render() {
+        return (
+            <Form className="upload-form" onSubmit={this.onSubmit}>
+                <Form.Label className="form-label" column={1}>Upload Your Work Here!</Form.Label>
+                <FilePond
+                    ref={ref => (this.pond = ref)}
+                    files={this.state.files}
+                    allowMultiple={true}
```

Comment:
Currently your upload only saves a single file

Reply:
ok, i want to leave it that way.